lib/edit_text_view.dart

```diff
@@ -1,32 +1,108 @@
+  EditTextViewState createState() {
+    return new EditTextViewState();
+  }
+}
+
+class EditTextViewState extends State<EditTextView> {
+  FocusNode myFocusNode = FocusNode();
+  bool viewtext = false;
+  String userTyped;
+  @override
+  void dispose() {
+    // Clean up the focus node when the Form is disposed
+    myFocusNode.dispose();
+
+    super.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return widget.fontType != null
+        ? !viewtext
+            ? LimitedBox(
+                child: Center(
+                    child: TextField(
+                        onSubmitted: (str) {
+                          myFocusNode.unfocus();
+                          setState(() {
+                            viewtext = true;
+                            userTyped = str;
+                          });
+                        },
+                        autofocus: true,
+                        focusNode: myFocusNode,
+                        textAlign: TextAlign.center,
+                        style: TextStyle(
+                            fontSize: 50.0,
+                            color: const Color(0xFF000000),
+                            fontWeight: FontWeight.bold,
+                            fontStyle: FontStyle.italic,
+                            fontFamily: widget.fontType),
+                        decoration: new InputDecoration.collapsed(
+                            hintText: widget.change))),
+              )
+            : DragTextView(fontType: widget.fontType, str: userTyped)
+        : Container();
+  }
+}
+
+class DragTextView extends StatefulWidget {
+  final String fontType;
+  final String str;
+
+  DragTextView({Key key, this.fontType, this.str}) : super(key: key);
+
+  @override
+  DragTextViewState createState() {
+    return new DragTextViewState();
+  }
+}
+
+class DragTextViewState extends State<DragTextView> {
+  Offset offset = Offset(0.0, 0.0);
+
   @override
   Widget build(BuildContext context) {
-    return fontType != null
-        ? Container(
-            height: 200.0,
-            child: Center(
-                child: Padding(
-              child: Center(
-                  child: TextField(
-                      textAlign: TextAlign.center,
-                      style: TextStyle(
-                          fontSize: 30.0,
-                          color: const Color(0xFF000000),
-                          fontWeight: FontWeight.bold,
-                          fontStyle: FontStyle.italic,
-                          fontFamily: fontType),
-                      decoration: new InputDecoration.collapsed(
-                          hintText: 'Type Here'))),
-              padding: EdgeInsets.all(20.0),
-            )),
-          )
-        : Text(" ");
+    // TODO: implement build
+    return Positioned(
+      top: offset.dy,
+      left: offset.dx,
+      child: GestureDetector(
+        child: Center(
+          child: LimitedBox(
+            child: Draggable(
```

Comment:
Do we need a Draggable strategy here? Seems like you need to move the entire widget, wherease Draggable is suitable where you have to use drag-and-drop